jax/experimental/jax2tf/tests/tf_test_util.py

```diff
@@ -111,14 +125,27 @@ def run_tf(mode):
       else:
         assert False
 
+    def is_tf_exception(lim: jax2tf.jax2tf.Limitation):
+      return (lim.ErrorType == 'Missing TF support' and
+              self.tf_default_device.device_type in lim.Devices)
+
     result_tf = None
     for mode in ("eager", "graph", "compiled"):
+      current_limitations = jax2tf.jax2tf.all_limitations[:]
```

Comment:
I am wondering if it makes sense to move `current_limitations` outside the loop. What if one of these executions has cached the jax2tf.convert? Then we do not see any limitations. 

At the same time, this gives me the idea that it may be helpful to document of the limitations still occur with `exeprimental_compile=True`, I suspect that many do not. Maybe that is for the next PR.

Reply:
This seems a bit tricky. If we are 100% sure that they should always have the same limitations, this'd make sense to move out `current_limitations` outside. Otherwise, as you point out, it may not be the case, and then I think this deserves some thinking.